src/protocols/hii_database.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright © 2019 Intel Corporation
+// Copyright (c) Microsoft Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
I don't add licensing information to the individual files, given that they are merely copied from the spec, anyway. So unless you insist, I would just drop this section. As mention elsewhere, I will gladly add the copyright information to `AUTHORS`.